e2e/e2eApi.ts

```diff
@@ -39,7 +39,22 @@ export class E2EApi {
     return (await this.count(selector)) > 0
   }
 
-  public clickOn = async (selector: string) => await (await this.getElement(selector)).click()
+  public clickOn = async (selector: string, x?: number, y?: number) => {
+    if (x === undefined && y === undefined) {
+      const elementHandle = await this.getElement(selector)
+      await elementHandle.click()
+
+      return
+    }
```

Comment:
as at this point it might be the case that either `x` or `y` is undefined, lets ensure that `undefined` value is coerced to `0`

Reply:
Introduced a separate `clickOnPosition()` method instead